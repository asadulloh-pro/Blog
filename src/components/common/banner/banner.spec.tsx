import React from "react";
import { render } from "@testing-library/react";
import Banner from "./banner.component";

it("renders Banner Slider component", () => {
  const { container } = render(<Banner image="/mock/banner.jpg" />);
  expect(container).toMatchSnapshot();
});
