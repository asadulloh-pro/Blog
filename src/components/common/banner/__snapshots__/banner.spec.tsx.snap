// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Banner Slider component 1`] = `
<div>
  <img
    alt="banner image"
    class="w-full max-h-[20.5rem] rounded-[0.75rem] object-cover object-center"
    data-nimg="1"
    decoding="async"
    height="100"
    loading="lazy"
    src="/_next/image?url=%2Fmock%2Fbanner.jpg&w=640&q=75"
    srcset="/_next/image?url=%2Fmock%2Fbanner.jpg&w=384&q=75 1x, /_next/image?url=%2Fmock%2Fbanner.jpg&w=640&q=75 2x"
    style="color: transparent;"
    width="300"
  />
</div>
`;
